translations/en.json

```diff
@@ -2709,5 +2709,6 @@
     "not-enough-assets": "Not enough assets to pay gas fees",
     "send-from-network" : "Send from {{network}}",
     "define-amount-sent-from-network" : "Define amount sent from {{network}} network",
-    "dont-auto-recalculate-network": "Don't auto recalculate {{network}}"
+    "dont-auto-recalculate-network": "Don't auto recalculate {{network}}",
+    "value-higher-than-send-amount": "This value is higher than entered send amount"
```

Comment:
this text looks a bit off, perhaps we revisit? 🤔 
i.e "This value is higher than entered send amount"

should be like?

"This value is higher than entered amount to send" ??